subsys/bluetooth/rpc/client/bt_rpc_gatt_client.c

```diff
@@ -988,3 +1060,479 @@ static void bt_gatt_cb_att_mtu_update_call_rpc_handler(CborValue *value, void *h
 NRF_RPC_CBOR_CMD_DECODER(bt_rpc_grp, bt_gatt_cb_att_mtu_update_call,
 			 BT_GATT_CB_ATT_MTU_UPDATE_CALL_RPC_CMD,
 			 bt_gatt_cb_att_mtu_update_call_rpc_handler, NULL);
+
+#if defined(CONFIG_BT_GATT_CLIENT)
+
+static size_t bt_gatt_discover_params_buf_size(const struct bt_gatt_discover_params *data)
+{
+	return bt_uuid_enc(NULL, data->uuid) + 3 + 3 + 3;
```

Comment:
why do we add these magic numbers?
is it auto-generated?

Reply:
It was auto-generated at first place, but I made some modifications later and I wanted to be sure that number is correct, so I spitted into parts as you see now. I can add them and put single number.

Reply:
I am okay with that if it's autogenerated. I think we should go back to the generated version so that we are sure later on that there were no functional modifications done to this section of code.

Reply:
Ok, so I changed to as it was in generator output.